### ISSUE TITLE
Linda not able to deal with a 10% slope

### ISSUE THREAD

**Issue by gestom** [strands-project/scitos_robot#27]
A ~4 meter long ramp with ~0.4m elevation has been installed in our place so Linda can access the other part of the offices. However, when we have tried to drive her up by joystick, her speed is gradually decreasing and she stops at ~1.2m of the ramp. Moreover, she cannot turn while moving forwards. According to the SCITOS-G5 flyer here: http://metralabs.com/images/ML_Downloads/Flyer_SCITOS_G5_Features_2.pdf, the robot should be able to deal with a 10% slope. 

@chmartin21 : Any solution for that ?

**Comment by chmartin21** [comment]
Do you have tried to increase the force? The force is a property of the "MainControlUnit" of the robot. You can start miracenter, increase the force using the PropertyView and then try it again. The default value is 80. But you can increase it up to 255. I would suggest to try something around 120 or 150. If you found the right value, you can put it in your configuration file using the parameter tag.

**Comment by gestom** [comment]
Works like a charm, thanks for advice.

**Comment by hawesie** [comment]
@chmartin21 should we all increase the force as a default, even if we're not tackling slopes?

**Comment by cdondrup** [comment]
Might also help to get rid of the stuck-in-carpet error.

**Comment by hawesie** [comment]
That's what I was thinking ;)

**Comment by cdondrup** [comment]
If I am not mistaken, we set the force parameter to 110 which enabled Linda to go up our 10% slope which is also covered in carpet. So it might be a good solution for you as well. If @chmartin21 sees no problem in increasing the force value for continuous usage, we can just open a pull request to alter the SCITOS config file.

I will reopen the issue until we have decided on opening a pull request or not.
